### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by mairaw**
Related to PR #116

### CODE REVIEW

File: snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs [hunk 1]
```diff
@@ -59,7 +59,7 @@ static byte[] encryptStringToBytes_AesManaged(string plainText, byte[] Key, byte
                 AesManagedAlg.Key = Key;
                 AesManagedAlg.IV = IV;
 
-                // Create a decrytor to perform the stream transform.
+                // Create an encrytor to perform the stream transform.
```
Comment (pkulikov) [inline snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs]: Typo: should be *encryptor*

File: snippets/csharp/VS_Snippets_CLR/tripledescryptoserviceprovider/cs/program.cs [hunk 2]
```diff
@@ -47,14 +47,15 @@ static byte[] EncryptStringToBytes(string plainText, byte[] Key, byte[] IV)
...
+            
             // Create an TripleDESCryptoServiceProvider object
             // with the specified key and IV.
             using (TripleDESCryptoServiceProvider tdsAlg = new TripleDESCryptoServiceProvider())
             {
                 tdsAlg.Key = Key;
                 tdsAlg.IV = IV;
 
-                // Create a decrytor to perform the stream transform.
+                // Create an encrytor to perform the stream transform.
```
Comment (pkulikov) [inline snippets/csharp/VS_Snippets_CLR/tripledescryptoserviceprovider/cs/program.cs]: Typo: should be *encryptor*

File: snippets/csharp/VS_Snippets_CLR/tripledescryptoserviceprovider/cs/program.cs [hunk 3]
```diff
@@ -102,7 +102,7 @@ static string DecryptStringFromBytes(byte[] cipherText, byte[] Key, byte[] IV)
...
```
Comment (pkulikov) [inline snippets/csharp/VS_Snippets_CLR/tripledescryptoserviceprovider/cs/program.cs]: Typo: should be *encryptor*

File: snippets/visualbasic/VS_Snippets_CLR/aessample/vb/program.vb [hunk 4]
```diff
@@ -46,20 +46,20 @@ Class AesExample
...
+        
         ' Create an Aes object
         ' with the specified key and IV.
         Using aesAlg As Aes = Aes.Create()
 
             aesAlg.Key = Key
             aesAlg.IV = IV
 
-            ' Create a decrytor to perform the stream transform.
+            ' Create an encrytor to perform the stream transform.
```
Comment (pkulikov) [inline snippets/visualbasic/VS_Snippets_CLR/aessample/vb/program.vb]: Typo: should be *encryptor*

File: snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs (hunk 1 above)
Comment (mairaw) [reply snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs]: Good catch! That’s what I had in mind to do. 😊

File: snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs [hunk 5]
```diff
@@ -59,7 +59,7 @@ static byte[] encryptStringToBytes_AesManaged(string plainText, byte[] Key, byte
                 AesManagedAlg.Key = Key;
                 AesManagedAlg.IV = IV;
 
-                // Create an encrytor to perform the stream transform.
+                // Create anencryptor to perform the stream transform.
```
Comment (pkulikov) [inline snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs]: @mairaw sorry I'm here again, but there should be a white space inside *anencryptor*
The same is true for other three updates.

Comment (mairaw) [reply snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs]: OMG! What’s wrong with me? 😂

Comment (pkulikov) [reply snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs]: Maybe, Sunday :)

File: snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs [hunk 6]
```diff
@@ -59,7 +59,7 @@ static byte[] encryptStringToBytes_AesManaged(string plainText, byte[] Key, byte
                 AesManagedAlg.Key = Key;
                 AesManagedAlg.IV = IV;
 
-                // Create a decrytor to perform the stream transform.
+                // Create anencryptor to perform the stream transform.
```
Comment (rpetrusha) [inline snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs]: anencryptor --> an encryptor

File: snippets/csharp/VS_Snippets_CLR/tripledescryptoserviceprovider/cs/program.cs [hunk 7]
```diff
@@ -47,14 +47,15 @@ static byte[] EncryptStringToBytes(string plainText, byte[] Key, byte[] IV)
...
+            
             // Create an TripleDESCryptoServiceProvider object
             // with the specified key and IV.
             using (TripleDESCryptoServiceProvider tdsAlg = new TripleDESCryptoServiceProvider())
             {
                 tdsAlg.Key = Key;
                 tdsAlg.IV = IV;
 
-                // Create a decrytor to perform the stream transform.
+                // Create anencryptor to perform the stream transform.
```
Comment (rpetrusha) [inline snippets/csharp/VS_Snippets_CLR/tripledescryptoserviceprovider/cs/program.cs]: anencryptor --> an encryptor

File: snippets/csharp/VS_Snippets_CLR/tripledescryptoserviceprovider/cs/program.cs [hunk 8]
```diff
@@ -102,7 +102,7 @@ static string DecryptStringFromBytes(byte[] cipherText, byte[] Key, byte[] IV)
...
```
Comment (rpetrusha) [inline snippets/csharp/VS_Snippets_CLR/tripledescryptoserviceprovider/cs/program.cs]: anencryptor --> an encryptor

File: snippets/visualbasic/VS_Snippets_CLR/aesmanaged/vb/program.vb [hunk 9]
```diff
@@ -46,20 +46,20 @@ Class AesExample
...
+        
         ' Create an AesManaged object
         ' with the specified key and IV.
         Using aesAlg As New AesManaged()
 
             aesAlg.Key = Key
             aesAlg.IV = IV
 
-            ' Create a decrytor to perform the stream transform.
+            ' Create a encryptor to perform the stream transform.
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_CLR/aesmanaged/vb/program.vb]: a --> an

File: snippets/visualbasic/VS_Snippets_CLR/aessample/vb/program.vb [hunk 10]
```diff
@@ -46,20 +46,20 @@ Class AesExample
...
+        
         ' Create an Aes object
         ' with the specified key and IV.
         Using aesAlg As Aes = Aes.Create()
 
             aesAlg.Key = Key
             aesAlg.IV = IV
 
-            ' Create a decrytor to perform the stream transform.
+            ' Create anencryptor to perform the stream transform.
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_CLR/aessample/vb/program.vb]: anencryptor --> an encryptor

**Comment by mairaw** [comment]
And you'd think this is an easy change 🤣 I think I finally got this right!

**Pull request merged**